webservices/utils.py

```diff
@@ -44,9 +44,9 @@ class Resource(six.with_metaclass(MethodResourceMeta, restful.Resource)):
 
 
 API_KEY_ARG = fields.Str(
-    required=True, missing="DEMO_KEY", description=docs.API_KEY_DESCRIPTION,
+     missing="DEMO_KEY", description=docs.API_KEY_DESCRIPTION,
```

Comment:
required = True caused an error when pushing to PROD. I was not able to reproduce this locally. But I tested using just missing and it worked on Stage. 

I think the error is due to a change in webargs where multiple calls to use_kwargs causes a bug when one of them is required. Here is a comment mentioning [this](https://github.com/marshmallow-code/webargs/issues/483#issuecomment-606811307). Our situation is not exactly the same, but I think it's the same error.